hbase-server/src/main/java/org/apache/hadoop/hbase/master/replication/ReplicationPeerManager.java

```diff
@@ -501,6 +511,32 @@ private void checkClusterKey(String clusterKey) throws DoNotRetryIOException {
     }
   }
 
+  private void checkClusterId(String clusterKey) throws DoNotRetryIOException {
+    String peerClusterId = "";
+    try {
+      // Create the peer cluster config for get peer cluster id
+      Configuration peerConf = HBaseConfiguration.createClusterConf(conf, clusterKey);
+      ZKWatcher zkWatcher =
+        new ZKWatcher(peerConf, this + "check-peer-cluster-key", new Abortable() {
```

Comment:
Looks like ZKWatcher is null-safe with abortable. Since we are not actually relying on it, just pass null?